src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/HdcVariationLicence.kt

```diff
@@ -0,0 +1,512 @@
+    probationPduDescription: String? = this.probationPduDescription,
+    probationLauCode: String? = this.probationLauCode,
+    probationLauDescription: String? = this.probationLauDescription,
+    probationTeamCode: String? = this.probationTeamCode,
+    probationTeamDescription: String? = this.probationTeamDescription,
+    appointmentPersonType: AppointmentPersonType? = this.appointmentPersonType,
+    appointmentPerson: String? = this.appointmentPerson,
+    appointmentTime: LocalDateTime? = this.appointmentTime,
+    appointmentTimeType: AppointmentTimeType? = this.appointmentTimeType,
+    appointmentAddress: String? = this.appointmentAddress,
+    appointmentContact: String? = this.appointmentContact,
+    spoDiscussion: String? = this.spoDiscussion,
+    vloDiscussion: String? = this.vloDiscussion,
+    approvedDate: LocalDateTime? = this.approvedDate,
+    approvedByUsername: String? = this.approvedByUsername,
+    approvedByName: String? = this.approvedByName,
+    supersededDate: LocalDateTime? = this.supersededDate,
+    submittedDate: LocalDateTime? = this.submittedDate,
+    dateCreated: LocalDateTime? = this.dateCreated,
+    dateLastUpdated: LocalDateTime? = this.dateLastUpdated,
+    updatedByUsername: String? = this.updatedByUsername,
+    variationOfId: Long? = this.variationOfId,
+    licenceVersion: String? = this.licenceVersion,
+    standardConditions: List<StandardCondition> = this.standardConditions,
+    additionalConditions: List<AdditionalCondition> = this.additionalConditions,
+    bespokeConditions: List<BespokeCondition> = this.bespokeConditions,
+    responsibleCom: CommunityOffenderManager? = this.responsibleCom,
+    curfewTimes: List<HdcCurfewTimes> = this.curfewTimes,
+    submittedBy: CommunityOffenderManager? = this.submittedBy,
+    createdBy: CommunityOffenderManager? = this.createdBy,
+    updatedBy: Staff? = this.updatedBy,
+  ): HdcVariationLicence {
+    val hdcVariationLicence: HdcVariationLicence = HdcVariationLicence(
+      id = id,
+      typeCode = typeCode,
+      version = version,
+      statusCode = statusCode,
+      nomsId = nomsId,
+      bookingNo = bookingNo,
+      bookingId = bookingId,
+      crn = crn,
+      pnc = pnc,
+      cro = cro,
+      prisonCode = prisonCode,
+      prisonDescription = prisonDescription,
+      prisonTelephone = prisonTelephone,
+      forename = forename,
+      middleNames = middleNames,
+      surname = surname,
+      dateOfBirth = dateOfBirth,
+      conditionalReleaseDate = conditionalReleaseDate,
+      actualReleaseDate = actualReleaseDate,
+      sentenceStartDate = sentenceStartDate,
+      sentenceEndDate = sentenceEndDate,
+      licenceStartDate = licenceStartDate,
+      licenceExpiryDate = licenceExpiryDate,
+      licenceActivatedDate = licenceActivatedDate,
+      homeDetentionCurfewActualDate = homeDetentionCurfewActualDate,
+      homeDetentionCurfewEndDate = homeDetentionCurfewEndDate,
```

Comment:
`homeDetentionCurfewActualDate` and `homeDetentionCurfewEndDate` hdc specific fields